### ISSUE TITLE
Keyhive UT47.2 support outdated

### ISSUE THREAD

**Issue by Jotil** [the-via/keyboards#1704]
tl;dr VIA client shows empty Lighting menu. v3 folder in repo doesn't have JSON.

I've been using VIA on my UT47.2 since last year. Innitially when I first set it up, everything worked fine with all the lighting menus apearing properly. Recently though I noticed that the Lighting menu is there but is empty [attached screenshot].

![image](https://user-images.githubusercontent.com/16452145/219854214-6d56f925-0936-4e09-af8b-6d6ddffab606.png)

I checked into the "v3" folder in this repo, and there doesn't seem to be anything related to keyhive/ut472 anywhere. I also tried making my own JSON with the v3 menu code `  "menus": ["qmk_rgblight"], ` and it brings back the underglow options but I'm not much of a programmer/git user so I donno if what I did is correct or not. 

Assistance would be appriciated.

Edit: I tried my hand at uploading the JSON file in a fork, but idk if I did it right...
[v3/ut472/ut472.json](https://github.com/Jotil/keyboards/blob/4109d42861dfa34e0e139d13947ce565f93769c0/v3/ut472/ut472.json)

**Comment by wilba** [comment]
Thanks for the bug report - there were V2 and V3 definitions in `ai03/ut472/ut472.json` not in a `keyhive` directory. They had `qmk_backlight_rgblight` and should have been `qmk_rgblight`. The firmware was failing to respond to the backlight commands hence the lack of controls. It should be fixed now if you reload VIA.

**Comment by Jotil** [comment]
Aha! I didn't think about looking into ai03's folder. Thanks a lot for the quick fix @wilba !! Works properly again.
